--- bindings/python/CMakeLists.txt	2026-03-18 19:33:50.111279184 +0530
+++ bindings/python/CMakeLists.txt	2026-03-18 19:33:50.160668136 +0530
@@ -65,7 +65,5 @@
 	COMMAND ${Python_EXECUTABLE} -m build
 	WORKING_DIRECTORY "${CMAKE_BINARY_DIR}"
 	DEPENDS ${DEPS})
-add_custom_target(python ALL DEPENDS smu ${OUTPUT})
+add_custom_target(python DEPENDS smu ${OUTPUT})
 
-install(CODE "execute_process(WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
-	COMMAND ${Python_EXECUTABLE} ${SETUP_PY} install --root=\$ENV{DESTDIR}/ --prefix=${CMAKE_INSTALL_PREFIX})")
